src/Sulu/Bundle/CategoryBundle/Twig/CategoryTwigExtension.php

```diff
@@ -74,7 +74,7 @@ public function getFunctions()
      * If parentKey is set, only the children of the category which is assigned to the given key are returned.
      *
      * @param string $locale
-     * @param string $parentKey key of parent category
+     * @param int|string $parentKey key of parent category
```

Comment:
this looks unexpected as a key should be a string. also findChilrenByParentKey accept string|null not int. only findChildrenByParentId accept int but that is not used here.

Reply:
In the tests the "key" is the id of the entity. But then I'll just fix the tests.

Reply:
yeah would adopt the tests then that they use a proper string key instead of int.